taskcluster/ffios_taskgraph/job.py

```diff
@@ -57,6 +57,7 @@ def configure_run_commands_schema(config, job, taskdesc):
 
 def _generate_secret_command(secret):
     secret_command = [
+        "python3",  # XXX Other mobile projects run this script under python2
```

Comment:
As far as I know, there is nothing to worry about today. I just left a reminder in case we hit a weird python3 issue someday.